sigpac_tools/utils.py

```diff
@@ -0,0 +1,91 @@
+import math
+import pyproj
+
+from sigpac_tools._globals import PROVINCES_BY_COMMUNITY
+
+
+def lng_lat_to_tile(lng: float, lat: float, zoom: float) -> tuple[int, int]:
+    """Transforms the given coordinates from longitude and latitude to tile coordinates for the given zoom level
+
+    Parameters
+    ----------
+    lng : float
+        Longitude of the location
+    lat : float
+        Latitude of the location
+    zoom : float
+        Zoom level to get the tile coordinates
+    
+    Returns
+    -------
+    tuple[int, int]
+        Returns a tuple with the x and y tile coordinates
+    """
+    ORIGIN_SHIFT = 2 * math.pi * 6378137 / 2.0
```

Comment:
Add here a link to the "original" implementation on JS so if anyone comes here in the future, knows were all this magic numbers come from